v4/export/ir.go

```diff
@@ -19,6 +21,15 @@ type SQLRowIter interface {
 	HasNext() bool
 }
 
+type RowReceiverStringer interface {
+	RowReceiver
+	Stringer
+}
+
+type Stringer interface {
+	ToString() string
+}
```

Comment:
why not reuse `fmt.Stringer`?

Reply:
There is a naming collide between `fmt.Stringer`'s `String()` and `sql.NullString`'s `String` field.